controllers/database_reconciler_helpers.go

```diff
@@ -92,10 +98,14 @@ func (r *DatabaseReconciler) handleDelete(ctx context.Context, database *ndbv1al
 	} else if controllerutil.ContainsFinalizer(database, common.FINALIZER_DATABASE_SERVER) {
 		// Checking if the database instance still exists in NDB. (It might take some time for the delete db instance operation to complete)
 		// Proceed to delete the database server vm only after the database instance has been deleted.
+		r.recorder.Eventf(database, "Normal", DEPROVISIONING_COMPLETED_EVENT, "Database deprovisioned from NDB.")
+		r.recorder.Eventf(database, "Normal", DEPROVISIONING_STARTED_EVENT, "Deprovisioning database server from NDB.")
 		log.Info("Checking if database instance exists")
```

Comment:
Please move these events down after the db instance has been deprovisioned, NDB takes a couple of minutes to complete this operation.